org.eclipse.jdt.ls.debug/src/org/eclipse/jdt/ls/debug/adapter/AdapterUtils.java

```diff
@@ -208,106 +158,4 @@ public static String computeClassPath(IJavaProject javaProject) throws CoreExcep
         return classPath;
     }
 
-    /**
-     * Searches the source file from the fully qualified name and returns the uri string.
-     * @param project
-     *               project instance
-     * @param fullyQualifiedName
-     *               fully qualified name
-     * @return the uri string of source file
-     */
-    public static String getURI(IProject project, String fullyQualifiedName) {
```

Comment:
IProject is kind of jdt world, i think we should encapsulate all such jdt things and debug adpter shouldn't touch anything of jdt, so i removed it from the util